src/app/pages/OnDemandTvPage/OnDemandTvPage.tsx

```diff
@@ -112,7 +112,16 @@ const OnDemandTvPage = ({
     ? `${brandTitle} - ${episodeTitle} - ${brandName}`
     : headline;
 
-  const episodePageIdentifier = `${service}.bbc_${service}_tv.tv.${episodeId}.page`;
+  const mediaOverrides: MediaOverrides = {
```

Comment:
Let's leave this for now, but I wonder whether `onDemandTV/getInitialData` might be a better place for this logic.

Reply:
Cross-sharing below.

_We depend on useContext to derive the service
used to construct the identifier.
We can only reference useContext from a component AFAIK
hence the challenge of homing the logic in getInitialData._